deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -577,7 +577,11 @@ pub struct MessageData {
     pub file: Option<String>,
     pub location: Option<(f64, f64)>,
     pub override_sender_name: Option<String>,
+    /// Quoted message id. Takes preference over `quoted_text` (see below).
     pub quoted_message_id: Option<u32>,
+    /// If this is `Some((text, protect))`, `protect` specifies whether `text` should only be sent
+    /// encrypted. If it should, but the message is unencrypted, `text` is replaced with "...".
```

Comment:
what should the ui set for protected?

Reply:
Btw, for bots `protect` is also doubtful. If you tap on this "..." (in case of an unencrypted message), you won't jump to the quoted message, so there's no way to know what is quoted. Maybe better fail sending at all? But i'm afraid this can break the communication unnecessarily. CC @adbenitez

Reply:
for bots there is no use case for the protect parameter, since what is useful about setting a quote to "..." ?? the users will never know what is going on, if the chat is unencrypted and the bridged messages are encrypted they will never see any quote

Reply:
Removed this `protect` from the jsonrpc API. But it's still needed in Core